yt/frontends/arepo/data_structures.py

```diff
@@ -73,6 +73,10 @@ def _get_uvals(self):
         handle.close()
         if missing:
             uvals = None
+        else:
+            # We assume this is comoving, because in the absence of comoving
+            # integration the redshift will be zero.
+            self._unit_base["cmcm"] = 1.0 / self._unit_base["UnitLength_in_cm"]
```

Comment:
I think you want `uvals[]` and here ant not `self._unit_base[]`

Reply:
Doh, yes! Thanks!